jaimemendozadev-los-angeles.md

```diff
@@ -0,0 +1,17 @@
+##Los Angeles, CA
+
+### Restaurants
+
++ [Tsujita](https://www.yelp.com/biz/tsujita-la-artisan-noodle-los-angeles-2)
++ [In-N-Out Burger](http://www.in-n-out.com/)
++ [Langer's Delicatessen](http://www.langersdeli.com/)
++ [Cassell's Hamburgers](http://www.cassellshamburgers.com/)
+
+### Places to Go
++ Dodger Stadium
++ [The Getty](https://www.getty.edu/)
++ [Disneyland](https://disneyland.disney.go.com/destinations/disneyland/)
+
```

Comment:
LOL, only go to Disneyland once and never go back. They're raising the prices for the annual pass this year I think.